docker-compose/docker-compose.yml

```diff
@@ -194,3 +192,13 @@ services:
          done"
     depends_on:
       - compute
+
+  neon-test-extensions:
+    image: ${REPOSITORY:-neondatabase}/neon-test-extensions-v${PG_TEST_VERSION:-16}:${TAG:-latest}
+    entrypoint:
+      - "/bin/bash"
+      - "-c"
+    command:
+      - "while true; do sleep 60; done"
```

Comment:
It looks like it can stuck

Reply:
I've done this to prevent the termination of the container.
If there is a better way for this I'll gladly implement it

Reply:
Let's add some meaningful timeout then, like 10-20-30 minutes?

Reply:
fixed 4bd2d7ef149f7d4a4ae71deb7f8393e5b7f92a6a